modules/openideal_webhooks/src/OpenidealWebhooksPluginTypes.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Drupal\openideal_webhooks;
+
+/**
+ * Contains outgoing webhooks types.
+ */
+class OpenidealWebhooksPluginTypes {
+
+  /**
+   * Outgoing webhook types.
+   */
+  const SLACK = 'slack';
```

Comment:
How about if we move these constants to `WebhookAction` class?

Reply:
Done